### DISCUSSION TITLE
Create UI helper classes to address white space issues

### DISCUSSION THREAD

**Discussion by nicdford** [pods-framework/pods#6774] [Ideas]
**Is your feature request related to a problem? Please describe.**
UI's that have fixed width elements leave awkward white spaces at various screen widths. 
![image](https://user-images.githubusercontent.com/7292444/43668529-50b1ed0c-9732-11e8-8f1b-941904efdf48.png) @JoryHogeveen phil said you did most the css for this dfv grid view, is there a reason we **have** to use fixed widths? I was concerned maybe the draggability functionality requires a fixed width? 

**Describe the solution you'd like**
We could develop some UI helper classes (using flexbox or maybe the new css grid specs) to address the need for more flexible user interfaces in the ever-demanding responsive world.

**Comment by JoryHogeveen** [comment]
The fixed items where already there, I just "relocated" them for that tiles view.
At that time I thought the same thing.. "Why fixed" but didn't change it because I thought there were other UI items what could rely on that CSS.

**Comment by quasel** [comment]
let's do it :)
